### DISCUSSION TITLE
Which quantum-espresso output file can I use for training MLIP?

### DISCUSSION THREAD

**Discussion by LuckyParky** [MDIL-SNU/SIMPLE-NN_v2#116] [Q&A]
Hi, I could see that simply using OUTCAR file from the VASP code is working well for training MLIP.
Then, which output file from the Quantum-espresso code is recommended?

**Comment by JisuJung928** [comment]
Hi LuckyParky,

The output that contains the total energy, force, and stress would be fine.
If you want to test whether SIMPLE-NN can handle the Quantum-espresso output, try the toy code below.

```python
from ase import io

atoms = io.read("output", format="espresso-out", index=":")
print(atoms.get_potential_energy(force_consistent=True))
```

Best regards,
Jisu.

**Reply by LuckyParky** [reply]
Thanks Jisu,

Input setting "refdata_format:     'espresso-out'" successfully handled the output file containing total energy, force, and stress.
